dsmr_parser/obis_references.py

```diff
@@ -66,8 +69,48 @@
 ACTUAL_SWITCH_POSITION = r'^\d-\d:96\.3\.10.+?\r\n'
 VALVE_POSITION_GAS = r'^\d-\d:24\.4\.0.+?\r\n'
 
+# Multiple 'slaves' can be linked to the main device.
+# The type is reported on 24.1.0
+# Specifications are in EN 13757-3
+# For example: Water mater = 7, Gas meter = 3
+# Identifier is on 96.1.0 (in NL for ex) or
+# on 96.1.1 (in BE for ex)
+# The values are reported on 24.2.1
+# With an exception in Belgium for the GAS meter
+# Be aware that for the gas volume, another OBIS-code is published
+# than the one listed in section 7 of DSMR P1.
+# This is due to the fact that in Belgium the not-temperature
+# corrected gas volume is used while in the Netherlands,
+# the temperature corrected gas volume is used.
+MBUS1_DEVICE_TYPE = r'^\d-1:24\.1\.0.+?\r\n'
+MBUS1_EQUIPMENT_IDENTIFIER = r'^\d-1:96\.1\.0.+?\r\n'
+MBUS1_EQUIPMENT_IDENTIFIER_ALT = r'^\d-1:96\.1\.1.+?\r\n'
+MBUS1_VALVE_POSITION = r'^\d-1:24\.4\.0.+?\r\n'
+MBUS1_METER_READING = r'^\d-1:24\.2\.1.+?\r\n'
+MBUS1_METER_READING_ALT = r'^\d-1:24\.2\.3.+?\r\n'
```

Comment:
May be these can be generic matching all devices:

```suggestion
MBUS_DEVICE_TYPE = r'^\d-\d:24\.1\.0.+?\r\n'
MBUS_EQUIPMENT_IDENTIFIER = r'^\d-\d:96\.1\.0.+?\r\n'
MBUS_EQUIPMENT_IDENTIFIER_ALT = r'^\d-\d:96\.1\.1.+?\r\n'
MBUS_VALVE_POSITION = r'^\d-\d:24\.4\.0.+?\r\n'
MBUS_METER_READING = r'^\d-\d:24\.2\.1.+?\r\n'
MBUS_METER_READING_ALT = r'^\d-\d:24\.2\.3.+?\r\n'
```

The exiting references are generic too. The regex will simple give more matches if there are more devices.

Reply:
Afaik there can only be a specific regex per value you want to parse. Otherwise it will only match the first one.